src/parsemultipart.jl

```diff
@@ -1,17 +1,35 @@
+module MultiPartParsing
+
+import ..access_threaded, ..Request, ..Multipart, ..payload
+using ..Parsers
+
+export parse_multipart_form
+
 const CR_BYTE = 0x0d # \r
 const LF_BYTE = 0x0a # \n
 const DASH_BYTE = 0x2d # -
 const HTAB_BYTE = 0x09 # \t
 const SPACE_BYTE = 0x20
-const RETURN_BYTES = [CR_BYTE, LF_BYTE]
+const SEMICOLON_BYTE = UInt8(';')
+const CRLFCRLF = [CR_BYTE, LF_BYTE, CR_BYTE, LF_BYTE]
```

Comment:
Make into a tuple (allows the compiler to know it's length etc)? I don't think it is mutated anywhere?